src/Mod/Arch/ArchFloor.py

```diff
@@ -194,6 +276,13 @@ def execute(self,obj):
                         o.Proxy.execute(o)
 
     def addObject(self,child):
+        """Adds the object to the floor's group.
```

Comment:
In general, the docstring for functions and methods should be in imperative mood. "Add", "run", "execute", and not "Adds", "runs", "executes". Also not, "this method does bla bla".

Reply:
Resolved in be64eeef3aba035da3af0bf13fa43a1b4001ce24